release/models/wifi/openconfig-wifi-phy.yang

```diff
@@ -122,7 +128,7 @@ module openconfig-wifi-phy {
 
     leaf channel {
       type uint8 {
-        range "1..165";
+        range "1..233";
```

Comment:
Just double checking the nomenclature of "Primary 20MHz" channel still makes sense in WiFi 6E? IOW, the description of this leaf-node is still valid ("this represents the primary 20 when operating at 40/80/160...")

Reply:
Yes, there is still a notion of a 20 MHz primary channel in the 6 GHz frequency range.